API-8177 pagination described (#848)
livechat/livechat-public-docs: content/messaging/agent-chat-api/index.mdx

</Code>
</Section>

## Pagination

<Section>

<Text>

Pagination is a mechanism that allows splitting the database output into more manageable chunks of data.

> Basing on `limit` and `sort order` parameters pagination is able to decide how many records will be returned at once and if it should fetch oldest or newest data first.

Reviewer: ```suggestion
Based on the `limit` and `sort order` parameters, pagination is able to decide how many records will be returned at once and whether it should fetch the oldest or the latest data first.
```

> Any filters that could be applied, should be provided in first pagination request. In response we can get `next_page_id` and `previous_page_id` parameters.

Reviewer: ```suggestion
Any filters that could be applied should be provided in the very first pagination request. In the response, you'll get the `next_page_id` and `previous_page_id` parameters.
```

> Next request should be performed using one of these parameters as `page_id` depending on iterations direction - forward or backward.

Reviewer: ```suggestion
You should make the subsequent request using one of these parameters as `page_id`, depending on the direction of iteration: forward or backward.
```

> `filters`, `limit` and `sort_order` parameters can not be provided along with `page_id`.

Reviewer: ```suggestion
The `filters`, `limit`, and `sort_order` parameters can't be provided along with `page_id`.
```
